src/dpp/events/channel_delete.cpp

```diff
@@ -40,24 +40,24 @@ using namespace dpp;
  */
 void channel_delete::handle(discord_client* client, json &j, const std::string &raw) {
 	json& d = j["d"];
+	dpp::guild* g = nullptr;
 	dpp::channel* c = dpp::find_channel(snowflake_not_null(&d, "id"));
 	if (c) {
-		dpp::guild* g = dpp::find_guild(c->guild_id);
+		g = dpp::find_guild(c->guild_id);
```

Comment:
yes this is fine. it just means cd.deleting_guild is g. We arent passed the whole guild in the event so without a guild cache this will simply be a nullptr, cluster.h has many new @ warning docs for this.

Reply:
(this is because the guild id comes from the channel's cached value, which is all we're given)